Traceback (most recent call last):
  File "/Users/corrado/opt/anaconda3/envs/pymc_env/lib/python3.11/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/Users/corrado/opt/anaconda3/envs/pymc_env/lib/python3.11/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/corrado/opt/anaconda3/envs/pymc_env/lib/python3.11/site-packages/jupyter_core/utils/__init__.py", line 173, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/corrado/opt/anaconda3/envs/pymc_env/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/Users/corrado/opt/anaconda3/envs/pymc_env/lib/python3.11/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/Users/corrado/opt/anaconda3/envs/pymc_env/lib/python3.11/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/Users/corrado/opt/anaconda3/envs/pymc_env/lib/python3.11/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
model_logit = bmb.Model(formula, data, family="binomial", priors=priors)
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mTypeError[0m                                 Traceback (most recent call last)
Cell [0;32mIn[15], line 1[0m
[0;32m----> 1[0m model_logit [38;5;241m=[39m [43mbmb[49m[38;5;241;43m.[39;49m[43mModel[49m[43m([49m[43mformula[49m[43m,[49m[43m [49m[43mdata[49m[43m,[49m[43m [49m[43mfamily[49m[38;5;241;43m=[39;49m[38;5;124;43m"[39;49m[38;5;124;43mbinomial[39;49m[38;5;124;43m"[39;49m[43m,[49m[43m [49m[43mpriors[49m[38;5;241;43m=[39;49m[43mpriors[49m[43m)[49m

File [0;32m~/opt/anaconda3/envs/pymc_env/lib/python3.11/site-packages/bambi/models.py:168[0m, in [0;36mModel.__init__[0;34m(self, formula, data, family, priors, link, categorical, potentials, dropna, auto_scale, noncentered, center_predictors, extra_namespace)[0m
[1;32m    166[0m     design [38;5;241m=[39m remove_common_intercept(design)
[1;32m    167[0m [38;5;28;01melse[39;00m:
[0;32m--> 168[0m     design [38;5;241m=[39m [43mfm[49m[38;5;241;43m.[39;49m[43mdesign_matrices[49m[43m([49m
[1;32m    169[0m [43m        [49m[38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43mformula[49m[38;5;241;43m.[39;49m[43mmain[49m[43m,[49m[43m [49m[38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43mdata[49m[43m,[49m[43m [49m[43mna_action[49m[43m,[49m[43m [49m[38;5;241;43m1[39;49m[43m,[49m[43m [49m[43madditional_namespace[49m
[1;32m    170[0m [43m    [49m[43m)[49m
[1;32m    172[0m [38;5;28;01mif[39;00m design[38;5;241m.[39mresponse [38;5;129;01mis[39;00m [38;5;28;01mNone[39;00m:
[1;32m    173[0m     [38;5;28;01mraise[39;00m [38;5;167;01mValueError[39;00m(
[1;32m    174[0m         [38;5;124m"[39m[38;5;124mNo outcome variable is set! [39m[38;5;124m"[39m
[1;32m    175[0m         [38;5;124m"[39m[38;5;124mPlease specify an outcome variable using the formula interface.[39m[38;5;124m"[39m
[1;32m    176[0m     )

File [0;32m~/opt/anaconda3/envs/pymc_env/lib/python3.11/site-packages/formulae/matrices.py:556[0m, in [0;36mdesign_matrices[0;34m(formula, data, na_action, env, extra_namespace)[0m
[1;32m    553[0m     [38;5;28;01melse[39;00m:
[1;32m    554[0m         [38;5;28;01mraise[39;00m [38;5;167;01mValueError[39;00m([38;5;124mf[39m[38;5;124m"[39m[38;5;124m'[39m[38;5;124mdata[39m[38;5;124m'[39m[38;5;124m contains [39m[38;5;132;01m{[39;00mincomplete_rows_n[38;5;132;01m}[39;00m[38;5;124m incomplete rows.[39m[38;5;124m"[39m)
[0;32m--> 556[0m design [38;5;241m=[39m [43mDesignMatrices[49m[43m([49m[43mdescription[49m[43m,[49m[43m [49m[43mdata[49m[43m,[49m[43m [49m[43menv[49m[43m)[49m
[1;32m    557[0m [38;5;28;01mreturn[39;00m design

File [0;32m~/opt/anaconda3/envs/pymc_env/lib/python3.11/site-packages/formulae/matrices.py:58[0m, in [0;36mDesignMatrices.__init__[0;34m(self, model, data, env)[0m
[1;32m     56[0m [38;5;28;01mif[39;00m [38;5;28mself[39m[38;5;241m.[39mmodel[38;5;241m.[39mresponse:
[1;32m     57[0m     [38;5;28mself[39m[38;5;241m.[39mresponse [38;5;241m=[39m ResponseMatrix([38;5;28mself[39m[38;5;241m.[39mmodel[38;5;241m.[39mresponse)
[0;32m---> 58[0m     [38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43mresponse[49m[38;5;241;43m.[39;49m[43mevaluate[49m[43m([49m[43mdata[49m[43m,[49m[43m [49m[43menv[49m[43m)[49m
[1;32m     60[0m [38;5;28;01mif[39;00m [38;5;28mself[39m[38;5;241m.[39mmodel[38;5;241m.[39mcommon_terms:
[1;32m     61[0m     [38;5;28mself[39m[38;5;241m.[39mcommon [38;5;241m=[39m CommonEffectsMatrix([38;5;28mself[39m[38;5;241m.[39mmodel[38;5;241m.[39mcommon_terms)

File [0;32m~/opt/anaconda3/envs/pymc_env/lib/python3.11/site-packages/formulae/matrices.py:137[0m, in [0;36mResponseMatrix.evaluate[0;34m(self, data, env)[0m
[1;32m    135[0m [38;5;28mself[39m[38;5;241m.[39mdata [38;5;241m=[39m data
[1;32m    136[0m [38;5;28mself[39m[38;5;241m.[39menv [38;5;241m=[39m env
[0;32m--> 137[0m [38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43mterm[49m[38;5;241;43m.[39;49m[43mset_type[49m[43m([49m[38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43mdata[49m[43m,[49m[43m [49m[38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43menv[49m[43m)[49m
[1;32m    138[0m [38;5;28mself[39m[38;5;241m.[39mterm[38;5;241m.[39mset_data()
[1;32m    139[0m [38;5;28mself[39m[38;5;241m.[39mkind [38;5;241m=[39m [38;5;28mself[39m[38;5;241m.[39mterm[38;5;241m.[39mterm[38;5;241m.[39mkind

File [0;32m~/opt/anaconda3/envs/pymc_env/lib/python3.11/site-packages/formulae/terms/terms.py:820[0m, in [0;36mResponse.set_type[0;34m(self, data, env)[0m
[1;32m    818[0m [38;5;28;01mdef[39;00m [38;5;21mset_type[39m([38;5;28mself[39m, data, env):
[1;32m    819[0m [38;5;250m    [39m[38;5;124;03m"""Set type of the response term."""[39;00m
[0;32m--> 820[0m     [38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43mterm[49m[38;5;241;43m.[39;49m[43mset_type[49m[43m([49m[43mdata[49m[43m,[49m[43m [49m[43menv[49m[43m)[49m

File [0;32m~/opt/anaconda3/envs/pymc_env/lib/python3.11/site-packages/formulae/terms/terms.py:433[0m, in [0;36mTerm.set_type[0;34m(self, data, env)[0m
[1;32m    431[0m     component[38;5;241m.[39mset_type(data)
[1;32m    432[0m [38;5;28;01melif[39;00m [38;5;28misinstance[39m(component, Call):
[0;32m--> 433[0m     [43mcomponent[49m[38;5;241;43m.[39;49m[43mset_type[49m[43m([49m[43mdata[49m[43m,[49m[43m [49m[43menv[49m[43m)[49m
[1;32m    434[0m [38;5;28;01melse[39;00m:
[1;32m    435[0m     [38;5;28;01mraise[39;00m [38;5;167;01mValueError[39;00m(
[1;32m    436[0m         [38;5;124m"[39m[38;5;124mCan[39m[38;5;124m'[39m[38;5;124mt set type on Term because at least one of the components [39m[38;5;124m"[39m
[1;32m    437[0m         [38;5;124mf[39m[38;5;124m"[39m[38;5;124mis of the unexpected type [39m[38;5;132;01m{[39;00m[38;5;28mtype[39m(component)[38;5;132;01m}[39;00m[38;5;124m.[39m[38;5;124m"[39m
[1;32m    438[0m     )

File [0;32m~/opt/anaconda3/envs/pymc_env/lib/python3.11/site-packages/formulae/terms/call.py:104[0m, in [0;36mCall.set_type[0;34m(self, data_mask, env)[0m
[1;32m     88[0m [38;5;250m[39m[38;5;124;03m"""Evaluates function and determines the type of the result of the call.[39;00m
[1;32m     89[0m 
[1;32m     90[0m [38;5;124;03mEvaluates the function call and sets the ``.kind`` property to ``"numeric"`` or[39;00m
[0;32m   (...)[0m
[1;32m    100[0m [38;5;124;03m    The environment where values and functions are taken from.[39;00m
[1;32m    101[0m [38;5;124;03m"""[39;00m
[1;32m    103[0m [38;5;28mself[39m[38;5;241m.[39menv [38;5;241m=[39m env[38;5;241m.[39mwith_outer_namespace({[38;5;241m*[39m[38;5;241m*[39mTRANSFORMS, [38;5;241m*[39m[38;5;241m*[39mENCODINGS})
[0;32m--> 104[0m x [38;5;241m=[39m [38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43mcall[49m[38;5;241;43m.[39;49m[43meval[49m[43m([49m[43mdata_mask[49m[43m,[49m[43m [49m[38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43menv[49m[43m)[49m
[1;32m    106[0m [38;5;28;01mif[39;00m is_numeric_dtype(x):
[1;32m    107[0m     [38;5;28mself[39m[38;5;241m.[39mkind [38;5;241m=[39m [38;5;124m"[39m[38;5;124mnumeric[39m[38;5;124m"[39m

File [0;32m~/opt/anaconda3/envs/pymc_env/lib/python3.11/site-packages/formulae/terms/call_resolver.py:272[0m, in [0;36mLazyCall.eval[0;34m(self, data_mask, env)[0m
[1;32m    269[0m args [38;5;241m=[39m [arg[38;5;241m.[39meval(data_mask, env) [38;5;28;01mfor[39;00m arg [38;5;129;01min[39;00m [38;5;28mself[39m[38;5;241m.[39margs]
[1;32m    270[0m kwargs [38;5;241m=[39m {name: arg[38;5;241m.[39meval(data_mask, env) [38;5;28;01mfor[39;00m name, arg [38;5;129;01min[39;00m [38;5;28mself[39m[38;5;241m.[39mkwargs[38;5;241m.[39mitems()}
[0;32m--> 272[0m [38;5;28;01mreturn[39;00m [43mcallee[49m[43m([49m[38;5;241;43m*[39;49m[43margs[49m[43m,[49m[43m [49m[38;5;241;43m*[39;49m[38;5;241;43m*[39;49m[43mkwargs[49m[43m)[49m

[0;31mTypeError[0m: 'TensorVariable' object is not callable

